### PR TITLE
Display an error message when add subcommand cannot be found

### PR DESCRIPTION
**Pull request by rawkode**
Hopefully helps with #21

### CODE REVIEW

File: stable/add.pony [hunk 1]
```diff
@@ -2,15 +2,17 @@ use "json"
...
     let kind = args(0)?
     let rest = args.slice(1)
     let prim =
       match kind
       | "github" => AddGithub
       | "local-git" => AddLocalGit
       | "local" => AddLocal
-      else error
+      else
+        log("Couldn't find command '" + kind + "'. Available commands are github, local-git, and local")
```
Comment (jemc) [inline stable/add.pony]: I'm going to split this message up over multiple lines.

**Review by jemc — APPROVED** [review]
Thanks!

**Comment by SeanTAllen** [comment]
@rawkode thanks! feel free to dive in on more. 😉

**Comment by rawkode** [comment]
@SeanTAllen Keep tagging the easy issues until I'm further acquainted :+1:

**Pull request merged**